src/components/ui/Calendar.tsx

```diff
@@ -29,7 +29,7 @@ const Calendar = ({ className, classNames, ...props }: CalendarProps) => {
         row: 'grid grid-cols-7',
         tbody: 'grow grid grid-rows-6',
         cell: 'text-sm relative select-none focus-within:relative focus-within:z-20',
```

Comment:
It seems like you changed color of each day on hover, but **the issue was actually aiming to change the color of them on selected state.**

So for example, we can set the selected state color for each day like this ⬇️ 
```tsx
 day: 'w-full h-full rounded-md flex justify-center items-center aria-selected:bg-primary aria-selected:text-white hover:bg-gray-light',
```
With this implementation, **we might want to also remove `day_selected` attribute because it's not necessary anymore.**

Reply:
Apologies for the confusion @kanta1207 and thanks for the help! I've made the requested changes.